File: frontend/models/MaritalStatus.ts
export enum MaritalStatus {
  SINGLE = "SINGLE",
  MARRIED = "MARRIED",
  DIVORCED = "DIVORCED",
  WIDOWED = "WIDOWED",
  SEPARATED = "SEPARATED",
  ENGAGED = "ENGAGED",
  RELATIONSHIP = "RELATIONSHIP",
  CIVILUNION = "CIVILUNION"
}

export const MaritalStatusDescriptions: { [key in MaritalStatus]: string } = {
  [MaritalStatus.SINGLE]: 'Solteiro',
  [MaritalStatus.MARRIED]: 'Casado',
  [MaritalStatus.DIVORCED]: 'Divorciado',
  [MaritalStatus.WIDOWED]: 'Viúvo',
  [MaritalStatus.SEPARATED]: 'Separado',
  [MaritalStatus.ENGAGED]: 'Noivo',
  [MaritalStatus.RELATIONSHIP]: 'Em um relacionamento',
  [MaritalStatus.CIVILUNION]: 'União civil',
};

File: src/main/java/com/ltsoftwaresupport/lthotel/model/MaritalStatus.java
package com.ltsoftwaresupport.lthotel.model;

public enum MaritalStatus {
    SINGLE("Solteiro"),
    MARRIED("Casado"),
    DIVORCED("Divorciado"),
    WIDOWED("Viúvo"),
    SEPARATED("Separado"),
    ENGAGED("Noivo"),
    RELATIONSHIP("Namorando"),
    CIVILUNION("União Civil");
	
	private String maritalStatus;
	
	MaritalStatus(String maritalStatus) {
		this.maritalStatus = maritalStatus;
	}
	
	public String getMaritalStatus() {
		return maritalStatus;
	}
	
	@Override
	public String toString() {
		return this.getMaritalStatus();
	}
}